src/Raven.Studio/wwwroot/App/views/manage/editServerWideBackup.html

```diff
@@ -41,6 +41,9 @@
                         <ul class="dropdown-menu" data-bind="foreach: backupOptions">
                             <li><a href="#" data-bind="text: $data, click: $parent.useBackupType.bind($parent, $data)"></a></li>
                         </ul>
+                        <div class="bg-info padding-left-xxs" data-bind="visible: isSnapshot">
```

Comment:
Does it mean that backup won't be created or it will be created as non-snapshot? Maybe we should mention about that on UI.